Gruntfile.js

```diff
@@ -153,6 +153,7 @@ module.exports = function (grunt) {
                 '/' + staticDir + '/geo.ext.min.js',
                 // '/' + rootStaticDir + '/libs.min.js', // libs included in jade template
                 '/' + staticDir + '/jquery.gridster.js',
+                '/' + staticDir + '/jquery-ui.min.js',
```

Comment:
Yes, the files have to be moved under the static web root to be served by cherrypy.